resources/services/qldb/ledgers.go

```diff
@@ -0,0 +1,345 @@
+				Resolver:    schema.PathResolver("EncryptionDescription.InaccessibleKmsKeyDateTime"),
+			},
+			{
+				Name:        "name",
+				Description: "The name of the ledger.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "permissions_mode",
+				Description: "The permissions mode of the ledger.",
+				Type:        schema.TypeString,
+			},
+			{
+				Name:        "state",
+				Description: "The current status of the ledger.",
+				Type:        schema.TypeString,
+			},
+		},
+		Relations: []*schema.Table{
+			{
+				Name:        "aws_qldb_ledger_journal_kinesis_streams",
+				Description: "Information about an Amazon QLDB journal stream, including the Amazon Resource Name (ARN), stream name, creation time, current status, and the parameters of the original stream creation request.",
+				Resolver:    fetchQldbLedgerJournalKinesisStreams,
+				Columns: []schema.Column{
+					{
+						Name:        "ledger_cq_id",
+						Description: "Unique CloudQuery ID of aws_qldb_ledgers table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "stream_arn",
+						Description: "The Amazon Resource Name (ARN) of the Kinesis Data Streams resource.  This member is required.",
+						Type:        schema.TypeString,
+						Resolver:    schema.PathResolver("KinesisConfiguration.StreamArn"),
+					},
+					{
+						Name:        "aggregation_enabled",
+						Description: "Enables QLDB to publish multiple data records in a single Kinesis Data Streams record, increasing the number of records sent per API call",
+						Type:        schema.TypeBool,
+						Resolver:    schema.PathResolver("KinesisConfiguration.AggregationEnabled"),
+					},
+					{
+						Name:        "ledger_name",
+						Description: "The name of the ledger.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "role_arn",
+						Description: "The Amazon Resource Name (ARN) of the IAM role that grants QLDB permissions for a journal stream to write data records to a Kinesis Data Streams resource.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "status",
+						Description: "The current state of the QLDB journal stream.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "stream_id",
+						Description: "The UUID (represented in Base62-encoded text) of the QLDB journal stream.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "stream_name",
+						Description: "The user-defined name of the QLDB journal stream.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "arn",
+						Description: "The Amazon Resource Name (ARN) of the QLDB journal stream.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "creation_time",
```

Comment:
not `creation_date_and_time` like the parent resource? inconsistent

Reply:
That is how its in the original struct